apps/portals/cancercomplexity/src/config/routesConfig.ts

```diff
@@ -474,68 +507,7 @@ const routes: GenericRoute[] = [
           {
             path: 'DetailsPage',
             exact: false,
-            synapseConfigArray: [
```

Comment:
Moved to datasets.ts, and added one helpText.  Attempted to move others, but I ran into cyclical dependencies